entries/m/migros.ch.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Migros": {
+    "passwordless": "allowed",
+    "mfa": "allowed",
+    "categories": "retail",
+    "documentation": "https://account.migros.ch/help#faqPasskeyPrio",
+    "recovery": "https://account.migros.ch/help#faqPasskeyPrio",
```

Comment:
```suggestion
```

This is not a recovery documentation page. Per [2FA Directory's `CONTRIBUTING.md`](https://github.com/2factorauth/twofactorauth/blob/master/CONTRIBUTING.md#other-properties):
> The recovery field can be used to link to account recovery documentation about what to do if you lose access to your 2FA method.